tests/test_mute_alarms.py

```diff
@@ -0,0 +1,69 @@
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+import types
+import unittest
+
+from lsst.ts.standardscripts import BaseScriptTestCase, get_scripts_dir
+from lsst.ts.standardscripts.mute_alarms import MuteAlarms
+
+
+class TestMuteAlarms(BaseScriptTestCase, unittest.IsolatedAsyncioTestCase):
+    async def basic_make_script(self, index):
+        self.script = MuteAlarms(index=index)
+
+        return (self.script,)
+
+    async def test_configure(self):
+        async with self.make_script():
+            config = types.SimpleNamespace(
+                name=".*",
+                mutedBy="test",
+                duration=10,
+                severity="Warning",
+            )
+            await self.script.configure(config)
+            pass
+
+    async def test_run(self):
+        """
+        Test that the script is paused for the selected queue.
+        """
+        async with self.make_script():
+            # Configure the script
+            await self.configure_script(
+                name=".*",
+                mutedBy="test",
+                duration=10,
+                severity="WARNING",
+            )
+
+            # Run the script
+            await self.run_script()
+
```

Comment:
Here make an assertion that the `mute` command was awaited with the expected set of parameters:

```
self.script.watcher.cmd_mute.set_start.assert_awaited_with(.....)
```